src/components/Modal/ModalBasic/ModalMain.tsx

```diff
@@ -0,0 +1,9 @@
+import ModalShareBtns from "./ModalShareBtns";
+
+interface ModalMain {
+  shareButtons?: boolean;
+}
+
+export default function ModalMain({ shareButtons }: ModalMain) {
+  return <>{shareButtons && <ModalShareBtns />}</>;
+}
```

Comment:
이렇게 표현해주면 더 명확할 것 같아요~
```suggestion
export default function ModalMain({ shareButtons }: ModalMain) {
  if(!shareButtons) {
    return null;
  }
  return <ModalShareBtns />
}
```